### PR TITLE
Shortcut: Add Storybook stories

### PR DESCRIPTION
**Pull request by bangank36**
## What?

- Add missing story for[ Shortcut Component](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/shortcut)
- Remove `@example` syntax on JSDoc for these components so Storybook can parse properly https://github.com/WordPress/gutenberg/pull/53379#issuecomment-1667472444

## Why?

Missing stories for components has been discussed on https://github.com/WordPress/gutenberg/issues/36128#issuecomment-1650618499

## How?

- Add new /stories folder on /shortcut
- Remove `@example` syntax on the noted components
- About `Text` component: its stories and type migration should be addressed on followed-up PRs

## Testing Instructions

- Checkout this branch
- Run `npm run storybook:dev`
    1. Confirm the shortcut stories at `?path=/docs/components-shortcut--docs` is working
    2. Confirm [these stories](https://github.com/WordPress/gutenberg/pull/53379#issuecomment-1667472444) has the correct JSDoc on storybook

## Screenshots or screencast 
image

### CODE REVIEW

File: packages/components/src/shortcut/stories/index.story.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const meta: Meta< typeof Shortcut > = {
+	component: Shortcut,
+	title: 'Components/Shortcut',
+	argTypes: {},
+	parameters: {
+		controls: {
+			expanded: true,
+		},
+		docs: { source: { state: 'open' } },
```
Comment (mirka) [inline packages/components/src/shortcut/stories/index.story.tsx]:
```suggestion
	parameters: {
		controls: {
			expanded: true,
		},
		docs: { canvas: { sourceState: 'shown' } },
```

File: packages/components/CHANGELOG.md [hunk 2]
```diff
@@ -18,7 +18,9 @@
 
 -   `ControlGroup`, `FormGroup`, `ControlLabel`, `Spinner`: Remove unused `ui/` components from the codebase ([#52953](https://github.com/WordPress/gutenberg/pull/52953)).
 -   `MenuItem`: Convert to TypeScript ([#53132](https://github.com/WordPress/gutenberg/pull/53132)).
+-   `MenuItem`: Add Storybook stories ([#53613](https://github.com/WordPress/gutenberg/pull/53613)).
```
Comment (mirka) [inline packages/components/CHANGELOG.md]: Let's move this to the other PR.

**Review by mirka — COMMENTED** [review]
Looks good! Thanks for the `@example` cleanup too.

File: packages/components/CHANGELOG.md (hunk 2 above)
Comment (bangank36) [reply packages/components/CHANGELOG.md]: Sorted

**Review by mirka — APPROVED** [review]
Looks good, thank you!

**Pull request merged**